docs/contracts/v4/concepts/06-fees.mdx

```diff
@@ -0,0 +1,140 @@
+---
+id: fees
+title: Fees
+sidebar_position: 5
+---
+
+# Background on Fees
+In a liquidity pool, users can provide liquidity by depositing tokens. These positions can be modified
+by either adding or removing liquidity. Fees are also involved in these modifications - collected from
+traders who swap tokens in the pool and distributed to liquidity providers.
```

Comment:
I think we want to distinguish the 3 types of fees in v4

1. Swap fees, fees paid by swappers to liquidity providers
2. Hook fees, optional fees collected by hooks. Implementation dependent
3. Protocol fees, fees collected from swap fees. Currently disabled but UNI token holders can vote to enable it